source/extensions/tracers/well_known_names.h

```diff
@@ -18,6 +18,8 @@ class TracerNameValues {
   const std::string Zipkin = "envoy.zipkin";
   // Dynamic tracer
   const std::string DynamicOt = "envoy.dynamic.ot";
+  // Datadog tracer
```

Comment:
Move before DynamicOt, since that's logically different from the "branded" tracers.